aspects/request_timeout_handler.go

```diff
@@ -0,0 +1,41 @@
+package aspects
+
+import (
+	"github.com/julienschmidt/httprouter"
+	"github.com/prebid/prebid-server/config"
+	"net/http"
+	"strconv"
+)
+
+func QueuedRequestTimeout(f httprouter.Handle, custHeaders config.CustomHeaders) httprouter.Handle {
+
+	return func(w http.ResponseWriter, r *http.Request, params httprouter.Params) {
+
+		reqTimeInQueue := r.Header.Get(custHeaders.RequestTimeInQueue)
+		reqTimeout := r.Header.Get(custHeaders.RequestTimeoutInQueue)
+
+		//If request timeout headers are not specified - process request as usual
+		if reqTimeInQueue == "" && reqTimeout == "" {
```

Comment:
Use || here instead of && as they both need to be valid for the timeout comparison to work.

Reply:
Done